.travis.yml

```diff
@@ -35,13 +35,16 @@ addons:
     packages:
       - libsodium-dev
       - g++-9
+      - sqlite3
 
 env:
   - PATH=$HOME/bin/:$PATH
 
 # No way to install the apt package, so...
 before_install:
   - if [ "$TRAVIS_OS_NAME" = "osx" ];   then brew install libsodium; fi
+  - if [ "$TRAVIS_OS_NAME" = "osx" ];   then brew install sqlite3; fi
+  - if [ "$TRAVIS_OS_NAME" = "osx" ];   then export PKG_CONFIG_PATH="/usr/local/opt/sqlite/lib/pkgconfig"; fi
```

Comment:
![Doesn't work](https://i.kym-cdn.com/entries/icons/original/000/024/153/soundsgood.jpg).
Use:
```yaml
env:
  - PKG_CONFIG_PATH=/usr/local/opt/sqlite/lib/pkgconfig
```

At the top level

Reply:
Ah I missed the env part.